old-tutorials-in-development/tutorials-in-development/R/dc-spatio-temporal-intro/05-geographic-vs-projected-CRS.Rmd

```diff
@@ -128,38 +129,57 @@ The **geographic WGS84 lat/long** `CRS` has an origin - (0,0) -
 located at the intersection of the
 Equator (0° latitude) and Prime Meridian (0° longitude) on the globe.
 
-```{r, geographic-WGS84, echo=FALSE, message=FALSE, results='hide'}
+NOTE: The code for this map can be found in the .R document that is available for 
+download at the bottom of this page!
+
+```{r, geographic-WGS84, echo=FALSE, message=FALSE, results='hide', fig.cap="World map plotted on an X, Y grid with the axes labeled "longitude" and "latitude" and untis in degrees. The origin (0,0) is at the center of the plot."}
```

Comment:
@donal-at-NEON: This code box has `echo=FALSE`, so only the map will show up, but I added load libraries and set wd in case someone downloads the data and wants to follow along

Reply:
Great idea - thank you!